# This message holds data from pressure sensors in robot's feet
# (FSR, Force Sensing Resistor). It is used to measure ZMP
# position.
#
# data - legs array (0 - left, 1 - right)
#        (also, you can use more legs, why not?)

geometry_msgs/Wrench[] data